ci/jenkins/README.md

```diff
@@ -1,22 +1,35 @@
 ## Antrea CI: Jenkins
 
 ### Reasons for Jenkins
-We have tests as Github Action but Jenkins allows tests running on a cluster of multiple nodes and offers better environment setup options.
+We have tests as Github Actions but Jenkins allows tests running on a cluster of
+multiple nodes and offers better environment setup options.
 
 ### List of Jenkins jobs
-* [e2e](https://github.com/vmware-tanzu/antrea/tree/master/test/e2e): end-to-end tests for Antrea.
-* conformance: community tests using sonobuoy, focusing on "Conformance", and skipping "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage", "sig-auth", "sig-api-machinery", "sig-apps" and "sig-node".
-* network policy: community tests using sonobuoy, focusing on "Feature:NetworkPolicy", and skipping "allow ingress access from updated pod" and "named port".
-Test "named port" will be supported soon, see issue [#122](https://github.com/vmware-tanzu/antrea/issues/122).  
-Test "allow ingress access from updated pod" fails because of a bug in the test definition, see issue [#85908](https://github.com/kubernetes/kubernetes/issues/85908).
+* [e2e](https://github.com/vmware-tanzu/antrea/tree/master/test/e2e): end-to-end
+  tests for Antrea.
+* conformance: community tests using sonobuoy, focusing on "Conformance", and
+  skipping "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli",
+  "sig-storage", "sig-auth", "sig-api-machinery", "sig-apps" and "sig-node".
+* network policy: community tests using sonobuoy, focusing on
+  "Feature:NetworkPolicy", and skipping the "allow ingress access from updated
+  pod" test. This test fails because of a bug in the test definition; see issue
+  [#85908](https://github.com/kubernetes/kubernetes/issues/85908).
```

Comment:
There are available releases having the fix now: v1.18.0-beta.0, specifying `--kube-conformance-image-version v1.18.0-beta.0` to sonobuoy can fix it. @antoninbas @lzhecheng

Reply:
Good idea. I have updated current jenkins with this and the code for jenkins on cloud.